sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -75,7 +75,7 @@ export class CertificatesClient {
     deleteCertificateContacts(options?: RequestOptions): Promise<Contacts>;
     deleteCertificateIssuer(issuerName: string, options?: RequestOptions): Promise<CertificateIssuer>;
     deleteCertificateOperation(name: string, options?: RequestOptions): Promise<CertificateOperation>;
-    getCertificate(name: string, version: string, requestOptions?: coreHttp.RequestOptionsBase): Promise<Certificate>;
+    getCertificate(name: string, version: string, options?: RequestOptions): Promise<Certificate>;
```

Comment:
This looks wrong to me. If we change the type here, won't we also have to change the implementation of getCertificate?

Reply:
Sigh, nevermind. GitHub auto-collapsed where you did that :p